core/domain/classifier_services.py

```diff
@@ -124,6 +124,57 @@ def classify_string_classifier_rule(state, normalized_answer):
 
     return None
 
+def train(exploration):
```

Comment:
OK, thanks @pranavsid98!

The project design is still in a bit of flux (though I think it's settling down) so perhaps the best thing to do for now is to wait a day or two until the ML milestones are more fully fleshed out, and then execute accordingly? (In the meantime, perhaps it might be worth finishing up #3406, if you're up for it!)